src/application/application.js

```diff
@@ -16,12 +14,6 @@ export default Application.extend({
     this.$body = $(document.body);
     this.layout = new LayoutView();
     this.layout.render();
-
-    this.listenTo(routerChannel, {
-      'before:enter:route' : this.onBeforeEnterRoute,
-      'enter:route'        : this.onEnterRoute,
-      'error:route'        : this.onErrorRoute
-    });
```

Comment:
Makes sense modifying [backbone-routing](https://github.com/thejameskyle/backbone-routing) for this task.

I'm re-reading the source and I intend to send a pull request soon.

For now, I do a roll back in these parts, and commit removing only the unused collection in `src/colors/edit/route.js` so it can be merged and wait until we update **backbone-routing**.